dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -1,17 +1,6 @@
 {% macro athena__drop_relation(relation) -%}
-  {% set rel_type = adapter.get_table_type(relation.schema, relation.table) %}
-  {%- if rel_type is not none %}
-    {%- if rel_type == 'table' %}
-      {%- do adapter.clean_up_table(relation.schema, relation.table) -%}
-    {%- endif %}
-    {% call statement('drop_relation', auto_begin=False) -%}
-      {%- if relation.type == 'view' -%}
-        drop {{ relation.type }} if exists {{ relation.render() }}
-      {%- else -%}
-        drop {{ relation.type }} if exists {{ relation.render_hive() }}
-      {% endif %}
-    {%- endcall %}
-  {%- endif %}
+  {%- do adapter.clean_up_table(relation.schema, relation.table) -%}
```

Comment:
Some consideration about this: 
* if the relation.type is a view we don't need to clean the s3 therefore clean_up_table must not be invoked. 
* I will still leave the check `rel_type is not none` in the macro to avoid to make not needed calls

Reply:
> Some consideration about this:
> 
>     * if the relation.type is a view we don't need to clean the s3 therefore clean_up_table must not be invoked.
I noticed that when running a view model the drop_relation macro is not called. Should there still be a check for view relation.type?
> 
>     * I will still leave the check `rel_type is not none` in the macro to avoid to make not needed calls
This is now included.